src/components/FindMovie/FindMovie.js

```diff
@@ -1,55 +1,106 @@
-import React from 'react';
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
 import './FindMovie.scss';
+import { getFilms } from '../../api/api';
 
 import { MovieCard } from '../MovieCard';
 import movies from '../../api/movies.json';
 
-export const FindMovie = () => (
-  <>
-    <form className="find-movie">
-      <div className="field">
-        <label className="label" htmlFor="movie-title">
-          Movie title
-        </label>
-
-        <div className="control">
-          <input
-            type="text"
-            id="movie-title"
-            placeholder="Enter a title to search"
-            className="input is-danger"
-          />
-        </div>
+export const FindMovie = ({ addMovie }) => {
+  const [movie, changeMovie] = useState(movies[0]);
+  const [title, changeTitle] = useState('');
+  const [error, changeError] = useState(false);
```

Comment:
```suggestion
  const [movie, setMovie] = useState(movies[0]);
  const [title, setTitle] = useState('');
  const [error, setError] = useState(false);
```
it's a common practice to use prefix `set` for state changing functions

Reply:
done